backend/support/cog_over.py

```diff
@@ -13,8 +14,15 @@
 COG_LIMIT = 50_000_000
 DA_THRESHOLD_FACTOR = 0.25
 
+FIRST_BASELINE_YEAR = 2019
+BASELINE_VALID_FOR_YEARS = 5
 
-def compute_cog_over(federal_awards, submission_status, auditee_ein, auditee_uei):
+DBKEY_TO_UEI_TRANSITION_YEAR = "2022"
+
+
+def compute_cog_over(
+    federal_awards, submission_status, auditee_ein, auditee_uei, audit_year
+):
```

Comment:
This comment is not directly related to the current change, but since the PR modifies this function, I’d like to take the opportunity to clarify some points and potentially simplify the logic for improved readability.
While reviewing this function, I noticed that it checks for None or empty awards at [line 36](https://github.com/GSA-TTS/FAC/pull/4489/files#diff-c25fd3c8576fba3f3c065573fc604b1501208e68bb87c074bb8daf3b46124d0dR36). For context, we compute cog_over as part of dissemination, and based on our current [cross-validations](https://github.com/GSA-TTS/FAC/blob/sk/cog_rebase/backend/audit/cross_validation/check_has_federal_awards.py#L4), we do not allow reports with no awards to proceed. This makes the check at line 36 seem unnecessary to me.
If anything, I believe it would be better to throw an exception here to indicate an issue with the report. Since I wasn’t part of the discussions that led to this logic, do you recall any context or conversations around this decision? I also noticed a similar check on [line 118](https://github.com/GSA-TTS/FAC/pull/4489/files#diff-c25fd3c8576fba3f3c065573fc604b1501208e68bb87c074bb8daf3b46124d0dL90).